tests/framework/unit_tests/utils/testInputData.py

```diff
@@ -32,7 +32,7 @@
 import Steps
 
 # write tex
-stepSpec = Steps.MultiRun.getInputSpecification()()
+stepSpec = Steps.MultiRun.MultiRun.getInputSpecification()()
```

Comment:
I think this should probably use the factory, as
```python
Steps.factory.getClass('MultiRun').getInputSpecification()()
```
on general principle, although I'm not sure we have a requirement that factories be used.

Reply:
Okay, change made.